### DISCUSSION TITLE
waline 登录 https 失效

### DISCUSSION THREAD

**Discussion by kaolengmian7** [walinejs/waline#1500] [Q&A] [answered]
点击登录按钮，提示 https 证书失效
![image](https://user-images.githubusercontent.com/29266358/195968943-1aa26384-0650-4852-a0e9-baecdd6678cb.png)
除登录外，其他 https 接口正常
![image](https://user-images.githubusercontent.com/29266358/195968970-67a1b7df-1bd9-4e06-a452-66518a834c17.png)
求大佬解答

**Comment by lizheming** [comment, marked as answer]
自签证书么？看起来就是证书有问题，自己根据错误信息查一下吧

![image](https://user-images.githubusercontent.com/424491/195969257-3b33c313-0f20-424c-805f-ec3a700683ee.png)
